cpp/stdlib.cc

```diff
@@ -16,6 +16,28 @@
 #include "cpp/core_pyerror.h"  // e_die for strftime
 #include "mycpp/runtime.h"
 
+#ifdef HAVE_DIRENT_H
```

Comment:
Hmm it seems like this block is from CPython?

I feel like it should be possible to just use POSIX, I would surprised if we can't list a directory with just POSIX, and ZERO `#ifdef`

Generally I want to minimize `#ifdef` as much as possible, since it explodes the testing matrix